.github/scripts/compile.sh

```diff
@@ -21,6 +21,32 @@
 
 set -euxo pipefail
 
+WORKING=$(pwd)
+
+#region libwebsockets
+wget ${LWS_SOURCE_URL} -O libwebsockets.tar.gz
+tar -zxf libwebsockets.tar.gz --one-top-level=lws-src --strip-components 1
+
+cmake -S "$WORKING/lws-src" -B "$WORKING/lws_build" \
```

Comment:
You wanted to put each -D build option in its own line for clarity ?

Reply:
If I was adding the code for the first time, then I'd do it this way. Now I am trying to minimize the number of changes in the PR. I am pretty much rewriting the whole script, but still, what can be preserved should be preserved.

I don't expect we'll be touching the lws options much.

Reply:
I do think that putting each -D option in its own line increases clarity and also makes it look uniform within the file

Reply:
done

Reply:
@ganeshmurthy done